htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -212,6 +212,10 @@ class CommandeFournisseur extends CommonOrder
 	public $origin_id;
 	public $linked_objects = array();
 
+	public $rang;
```

Comment:
There is no need for $rang on an order. This data exists for line only.

Reply:
correction done